translate/gast_to_js.py

```diff
@@ -0,0 +1,65 @@
+
+
+def list_to_csv_str(l):
+    s = ""
+    for i in l:
+        s += str(i) + ", "
+    
+    return s[:-2] # remove last comma and space
+
+
+def logStatement(gast):
+    arg_string = list_to_csv_str(gast["args"])
+    return "console.log(" + arg_string + ")"
+
+
+def value_to_str(val):
+    if type(val) == str:
```

Comment:
Where is `str` coming from?

Reply:
stored in gast. For example x= "hello" would store "hello" in gast which would then have to be converted out.